cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -134,9 +134,9 @@ def self.config(env)
             cookies: whitelisted_cookies
           }
         ],
-        # Remaining Pegasus paths are cached, and vary only on language and default cookies.
+        # Remaining Pegasus paths are cached, and vary only on language, country, and default cookies.
         default: {
-          headers: LANGUAGE_HEADER,
+          headers: LANGUAGE_HEADER + COUNTRY_HEADER,
```

Comment:
This is effectively the same as `WHITELISTED_HEADERS` but thought we might want to keep this separate in case that list grows in the future.